andela/codepirates-ah-frontend
# #167313371 Build out users follow each other

#### What does this PR do?
-It has user follow and unfollow each other feature working
#### Description of Task to be completed?
- added dynamic button for a user to follow and unfollow each other
#### How should this be manually tested?
Download or clone the tutorial project code from this [repo,](https://github.com/andela/codepirates-ah-frontend) in your terminal then git checkout ` ft-users-follow-each-other-167313371`, install all required NPM packages by running ` npm install ` from the command line in the project root folder (where the package.json is located) then after in localhost visit single articles then you will see a button for author of the articles then follow him
#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/167313371
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/16412410/68737669-cc865d80-05ec-11ea-853c-cd13bfe9062a.png)

![image](https://user-images.githubusercontent.com/16412410/68737692-df992d80-05ec-11ea-82da-f41e63957d06.png)
![image](https://user-images.githubusercontent.com/16412410/68737727-ffc8ec80-05ec-11ea-8fe4-9110949fb6f4.png)

#### Questions:
